PyInstaller/hooks/hook-itk.py

```diff
@@ -0,0 +1,7 @@
+from PyInstaller.utils.hooks import collect_data_files
+
+hiddenimports = ['new']
+
+# If ITK is pip installed, gets all the files.
```

Comment:
"If pip installed" is misleading since file are collected no matter ho it was installed. Please update the comment.

Reply:
I moved the comment to the top of the file and rephrased it. I also added a similar phrasing in the commit message. This note is to let the user know that in order for this hook to work, they need to use ITK installed from pip. Many ITK developers directly use ITK from its build tree, and this hook will not work when using that ITK installation.